.pfnci/run.sh

```diff
@@ -72,10 +72,9 @@ main() {
 
   "${PYTHON}" -m pip install /pfrl
   # TODO(pfrl): Prepare test target instead.
-  # pytest does not run with attrs==19.2.0 (https://github.com/pytest-dev/pytest/issues/3280)  # NOQA
   "${PYTHON}" -m pip install \
-      'pytest==4.1.1' 'attrs==19.1.0' 'pytest-xdist==1.26.1' \
-      'gym[atari,classic_control]==0.19.0' 'optuna' 'zipp==1.0.0' 'pybullet==2.8.1' 'jupyterlab==2.1.5'
+      'pytest==7.0.1' 'pytest-xdist==2.5.0' \
+      'gym[atari,classic_control]==0.19.0' 'optuna' 'zipp==1.0.0' 'pybullet==2.8.1' 'jupyterlab==3.3.1'
```

Comment:
What issue do these version changes try to solve? I confirmed that at least for the jupyter-related issue in pfn.cpu.examples CI can be resolved by #169 .

Reply:
Thanks. I tried to fix the jupyter issue. I'll revert the change to this file.